.github/workflows/nightly-sync.yml

```diff
@@ -0,0 +1,42 @@
+name: Nightly Sync of content
+
+on:
+  schedule:
+  - cron: '0 3 * * *'
+
+jobs:
+  checks:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+
+      - name: Cache node modules
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: Install
+        run: npm ci
+
+      - name: Collect latest releases
+        run: npm run collect
+
+      - name: Commit files
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+          git commit -am "Nightly Sync"
+
+      - name: Push changes
+        # Only run this on the main nodejs repo and not forks
+        if: github.repository == 'nodejs/i18n'
+        uses: ad-m/github-push-action@v0.5.0
```

Comment:
This configuration will expose our GitHub token to a third-party codebase. This looks like a legit Action and I don't intend to block this from shipping, but just want to point that out.

Reply:
Yup, didn't want to hand roll the push, but I don't think it is too much work.
It was the reason I pinned it to a particular release instead of `master`